lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -177,6 +177,7 @@ func (s *builderServicer) Build(ctx context.Context, request *builder_protos.Bui
 			FederatedModeMap:         getFederatedModeMap(federatedNetworkConfigs),
 			CongestionControlEnabled: swag.BoolValue(congestionControlEnabled),
 			SentryConfig:             getNetworkSentryConfig(&network),
+			EnableConvergedCore:      swag.BoolValue(gwEpc.EnableConvergedCore),
```

Comment:
Why have a network-wide config value if we only ever set the mconfig value based on the gateway-wide config value?

Either take network-wide into account, or remove the capacity to set network-wide in this PR

Reply:
Will take network-wide into account.